src/components/Audiences/index.js

```diff
@@ -23,11 +23,10 @@ const createAudiences = ({ config, logger }) => {
     lifecycle: {
       onBeforeEvent({ event, isViewStart }) {
         if (isViewStart) {
-          event.mergeQuery({
+          event.mergeMeta({
             activation: {
-              // TODO: Is this moving to a backend configuration?
-              urlsEnabled: config.urlDestinationsEnabled,
-              cookiesEnabled: config.cookieDestinationsEnabled
+              urlDestinationsEnabled: config.urlDestinationsEnabled,
+              storedDestinationsEnabled: config.cookieDestinationsEnabled
```

Comment:
Doesn't this potentially create third-party cookies? If so, shouldn't it also be tied to `config.thirdPartyCookiesEnabled` or maybe we should log a message of some sort if `config.cookieDestinationsEnabled` is `true` and `config.thirdPartyCookiesEnabled` is `false`? I haven't really thought through this.

Reply:
Those cookies can only be 1st party since they will be created by the JS on the page.